test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -396,6 +396,214 @@ def check_lpd(prototypes_base)
     end
   end
 
+  def check_hvac_sizing(prototypes_base)
+    prototypes_base.each do |prototype, model_baseline|
+      building_type, template, climate_zone, mod = prototype
+
+      # check sizing parameters (G3.1.2.2)
+      sizing_parameters = model_baseline.getSizingParameters
+      assert((sizing_parameters.coolingSizingFactor - 1.15).abs < 0.001, "Baseline cooling sizing parameters for the #{building_type}, #{template}, #{climate_zone} model is incorrect. The cooling sizing parameter is #{sizing_parameters.coolingSizingFactor} but should be 1.15")
+      assert((sizing_parameters.heatingSizingFactor - 1.25).abs < 0.001, "Baseline cooling sizing parameters for the #{building_type}, #{template}, #{climate_zone} model is incorrect. The heating sizing parameter is #{sizing_parameters.heatingSizingFactor} but should be 1.25")
+
+      # check sizing schedules for loads are correct (min/max) (G3.1.2.2.1 and exception)
+      check_sizing_values(model_baseline, building_type, template, climate_zone)
+
+      # check delta t between supply air temperature set point and room temperature set point are 20 deg (exception of 17 deg for laboratory spaces) (G3.1.2.8.1 and exception)
+      # including checking unit heater supply air temperature set point of 105 deg (G3.1.2.8.2)
+      check_sizing_delta_t(model_baseline, building_type, template, climate_zone)
+    end
+  end
+
+  def check_sizing_delta_t(model, building_type, template, climate_zone)
```

Comment:
We probably want to add the YARD comments before merging.

Reply:
done